### PR TITLE
Adding sql files to master

### PR DESCRIPTION
**Pull request by youngelvis**

### CODE REVIEW

File: Schema.sql [hunk 1]
```diff
@@ -0,0 +1,65 @@
+DROP DATABASE IF EXISTS `wavegod`;
+SET default_storage_engine=InnoDB;
+SET NAMES utf8mb4 COLLATE utf8mb4_unicode_ci;
+
+CREATE DATABASE IF NOT EXISTS wavegod
+    DEFAULT CHARACTER SET utf8mb4
+    DEFAULT COLLATE utf8mb4_unicode_ci;
+USE wavegod;
+
+create Table `Picture`(
```
Comment (ghost) [inline Schema.sql]: Please use UPPERCASE for all SQL specific key words such as CREATE, TABLE, FROM, SELECT etc

File: Schema.sql [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+    photoID int NOT NULL AUTO_INCREMENT,
+    photo varchar(40),
+    dateOfCreation datetime,
+    PRIMARY KEY (photoID)
+);
+
+create table `Fan` (
```
Comment (ghost) [inline Schema.sql]: Please use UPPERCASE for all SQL specific key words such as CREATE, TABLE, FROM, SELECT etc

File: Schema.sql [hunk 3]
```diff
@@ -0,0 +1,65 @@
...
+    country varchar(40) NOT NULL,
+    telephoneNo int NOT NULL,
+    dateOfBirth date NOT NULL,
+    dateOfCreation datetime NOT NULL,
+    photoID int NOT NULL,
+    PRIMARY KEY (fanID),
+    FOREIGN KEY (photoID) REFERENCES Picture(photoID)
+);
+
+create Table `Album`(
```
Comment (ghost) [inline Schema.sql]: Please use UPPERCASE for all SQL specific key words such as CREATE, TABLE, FROM, SELECT etc

File: Schema.sql [hunk 4]
```diff
@@ -0,0 +1,65 @@
...
+    albumID  int NOT NULL AUTO_INCREMENT, 
+    albumName  varchar(40) NOT NULL,
+    photoID int NOT NULL,
+    dateOfCreation  datetime NOT NULL,
+    isMixtape Boolean,
+    PRIMARY KEY (albumID),
+    FOREIGN KEY (photoID) REFERENCES Picture (photoID)
+);
+
+create Table `Track`(
```
Comment (ghost) [inline Schema.sql]: Please use UPPERCASE for all SQL specific key words such as CREATE, TABLE, FROM, SELECT etc

File: Schema.sql [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+    genre  varchar(20) NOT NULL,
+    Composer  varchar(40) NOT NULL,
+    featuredArtist varchar(40) NOT NULL,
+    albumID  int NOT NULL,
+    dateOfCreation datetime NOT NULL,
+    PRIMARY KEY (trackID),
+    FOREIGN KEY (albumID) REFERENCES Album (albumID)
+);
+
+create Table `Merchandise`(
```
Comment (ghost) [inline Schema.sql]: Please use UPPERCASE for all SQL specific key words such as CREATE, TABLE, FROM, SELECT etc

File: content.sql [hunk 6]
```diff
@@ -0,0 +1,15 @@
+
+SELECT *
+FROM Picture;
+
+select *
```
Comment (ghost) [inline content.sql]: Please use UPPERCASE for all SQL specific key words such as CREATE, TABLE, FROM, SELECT etc

File: insert.sql [hunk 7]
```diff
@@ -0,0 +1,90 @@
+insert into `Picture`(
```
Comment (ghost) [inline insert.sql]: Please use UPPERCASE for all SQL specific key words such as INSERT, CREATE, TABLE, FROM, SELECT etc

File: insert.sql [hunk 8]
```diff
@@ -0,0 +1,90 @@
...
+    
+    photo,
+    dateOfCreation
+)
```
Comment (Gbemiga) [inline insert.sql:5]: Missing ; and please insert space between this and next line.

File: insert.sql [hunk 9]
```diff
@@ -0,0 +1,90 @@
...
+values(
+    
+    'picture',
+    '2008-11-11 13:23:44'
+);
```
Comment (ghost) [inline insert.sql]: Insert space between this and next line.

File: insert.sql [hunk 10]
```diff
@@ -0,0 +1,90 @@
...
+    city,
+    country,
+    telephoneNo,
+    dateOfBirth,
+    dateOfCreation,
+    photoID
+   
+)
+values(
+
```
Comment (ghost) [inline insert.sql]: remove whitespace

File: insert.sql [hunk 11]
```diff
@@ -0,0 +1,90 @@
...
+    'dublin',
+    'ireland',
+    '0809343322',
+    '2008-11-11',
+    '2008-11-11 13:23:44',
+    '1'
+);
+
+insert into `Album`(
+
```
Comment (ghost) [inline insert.sql]: remove whitespace

File: insert.sql [hunk 12]
```diff
@@ -0,0 +1,90 @@
...
+insert into `Album`(
+    
+    albumName,
+    dateOfCreation,
+    isMixtape,
+    photoID
+   
+)
+values(
+
```
Comment (ghost) [inline insert.sql]: remove whitespace

File: insert.sql [hunk 13]
```diff
@@ -0,0 +1,90 @@
...
+values(
+    
+    'yaya',
+    '2008-11-11 13:23:44',
+    False,
+    '1'
+);
+
+insert into `Track`(
+
```
Comment (ghost) [inline insert.sql]: remove whitespace

**Review by ghost — CHANGES_REQUESTED** [review]
Please make requested changes PR

File: insert.sql (hunk 13 above)
Comment (youngelvis) [reply insert.sql]: done

File: insert.sql (hunk 12 above)
Comment (youngelvis) [reply insert.sql]: done

File: insert.sql (hunk 10 above)
Comment (youngelvis) [reply insert.sql]: done

File: Schema.sql (hunk 5 above)
Comment (youngelvis) [reply Schema.sql]: done

**Comment by ghost** [comment]
Please put all SQL files in an SQL folder.

**Comment by ghost** [comment]
@youngelvis Please put all SQL files in an SQL folder.

**Review by ghost — APPROVED** [review]
Okay looks good. Go ahead and merge 👍

**Comment by youngelvis** [comment]
next move?

**Pull request merged**